### ISSUE TITLE
Configuration issue with helm

### ISSUE THREAD

**Issue by riaanp78** [opencost/opencost-helm-chart#167]
I think I am not quite sure of how to configure the Google GKE JSON key.

In the helm chart I did this:

`cloudProviderApiKey: google-gke`

And the error on startup is this:

`opencost 2024-01-06T14:14:01.517584769Z INF No pricing-configs configmap found at install time, using existing configs: configmaps "pricing-configs" not f ││ opencost 2024-01-06T14:14:01.523173803Z INF No metrics-config configmap found at install time, using existing configs: configmaps "metrics-config" not fou ││ opencost 2024-01-06T14:14:01.524422233Z WRN Failed to load auth secret, or was not mounted: Secret does not exist                                          ││ opencost 2024-01-06T14:14:01.637512048Z ERR Failed to lookup reserved instance data: googleapi: Error 400: Invalid resource field value in the request.    ││ opencost Details:                                                                                                                                          ││ opencost [                                                                                                                                                 ││ opencost   {                                                                                                                                               ││ opencost     "@type": "type.googleapis.com/google.rpc.ErrorInfo",                                                                                          ││ opencost     "domain": "googleapis.com",                                                                                                                   ││ opencost     "metadatas": {                                                                                                                                ││ opencost       "method": "compute.v1.RegionCommitmentsService.AggregatedList",                                                                             ││ opencost       "service": "compute.googleapis.com"                                                                                                         ││ opencost     },                                                                                                                                            ││ opencost     "reason": "RESOURCE_PROJECT_INVALID"                                                                                                          ││ opencost   }                                                                                                                                               ││ opencost ]                                                                                                                                                 ││ opencost , invalidParameter                                                                                                                                ││ opencost 2024-01-06T14:15:46.705337891Z ERR Failed to load cluster info via query after 6 retries`

Any advice?

**Comment by mattray** [comment]
You'll want to use the API key following the instructions here: https://www.opencost.io/docs/configuration/gcp#configuring-gcp-pricing

**Comment by riaanp78** [comment]
@mattray I followed that exact example. What I was not sure is how to use that API key I generated.  Is the API key a secret name that contains cloud-integration.json or should the contents of cloud-integration.json be specified in the YAML config. This part is what is not working. But I have given up on this now. Very little examples on the internet on actually implementing this. We opted to just make a report using bigquery instead.
